app/components/login_layout/login_layout.module.css

```diff
@@ -1,9 +1,8 @@
 .container {
   display: grid;
   grid-template-columns: 1fr 1fr;
-  height: calc(100vh - 70px);
   width: 100vw;
-  background-image: var(--gradient-image), url('/images/background.webp');
```

Comment:
I think this is why at the authentication the background is white.

Reply:
I found it! and fix it in local.
@pabloverduzco  Do you have been fixed? or Can I push...?

Reply:
Go ahead, send the fix! 😄